.github/workflows/ci.yml

```
@@ -0,0 +1,11 @@
+name: CI
+
+on: push
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - run: bazelisk build //...
+      - run: bazelisk test //...
```

The GitHub Actions runners have Bazelisk installed with a `/usr/local/bin/bazel` symlink. `bazel` is Bazelisk:
https://github.com/actions/runner-images/blob/0021a7fb65ccad67c0dea16d808d734434ff7f05/images/linux/scripts/installers/bazel.sh#L12 
```suggestion
      - run: bazel test //...
```

---

btw, related to Bazelisk, you may want to change https://github.com/shabanzd/dependency-bazelizer/blob/f5964772fa9722b956ca3c414b7070378f722815/.bazelversion#L1 to `6.x` to use the latest Bazel 6?

---

rpath patching relies on the modules directory being of the format moduleName~moduleVersion, and this is not guaranteed by Bazel. For example, in the experimental bzlmod in version 5 the format was moduleName-moduleVersion. So I just chose a version I am sure of. But now that we are setting up CI, it will be fine to use `6.x`. Btw welcome onboard @lalten ! 🤩

---

changed to 6.x